plugins/arcgis/web-app/projects/main/src/lib/arc-admin/arc-admin.component.html

```diff
@@ -3,80 +3,88 @@ <h1>ArcGIS Configuration</h1>
+          <div class="edit-startup-interval-field">
+            <mat-form-field appearance="fill">
+              <mat-label>Startup Interval</mat-label>
+              <input type="number" min="0" matInput value="{{editConfig.startupIntervalSeconds}}"
+                (keyup)="setNumberField('startupIntervalSeconds', startupIntervalSecondsField.value, 0)"
+                (input)="setNumberField('startupIntervalSeconds', startupIntervalSecondsField.value, 0)"
+                #startupIntervalSecondsField />
+                <mat-hint>Startup interval in seconds to wait for feature layer processors to be ready</mat-hint>
+            </mat-form-field>
+          </div>
+          <div class="edit-update-interval-field">
+            <mat-form-field appearance="fill">
+              <mat-label>Update Interval</mat-label>
+              <input type="number" min="0" matInput value="{{editConfig.updateIntervalSeconds}}"
+                (keyup)="setNumberField('updateIntervalSeconds', updateIntervalSecondsField.value, 0)"
+                (input)="setNumberField('updateIntervalSeconds', updateIntervalSecondsField.value, 0)"
+                #updateIntervalSecondsField />
+                <mat-hint>Processing wait time interval in seconds when pending observation updates exist</mat-hint>
+            </mat-form-field>
+          </div>
+          <div class="batch-size-field">
+            <mat-form-field appearance="fill">
+              <mat-label>Batch Size</mat-label>
+              <input type="number" min="1" matInput value="{{editConfig.batchSize}}"
+                (keyup)="setNumberField('batchSize', batchSizeField.value, 1)"
+                (input)="setNumberField('batchSize', batchSizeField.value, 1)" #batchSizeField />
+                <mat-hint>Maximum number of observations to process during a single time interval</mat-hint>
+            </mat-form-field>
+          </div>
+          <div class="attachment-modified-tolerance-field">
+            <mat-form-field appearance="fill">
+              <mat-label>Attachment Tolerance</mat-label>
+              <input type="number" min="0" matInput value="{{editConfig.attachmentModifiedTolerance}}"
+                (keyup)="setNumberField('attachmentModifiedTolerance', attachmentModifiedToleranceField.value, 0)"
+                (input)="setNumberField('attachmentModifiedTolerance', attachmentModifiedToleranceField.value, 0)"
+                #attachmentModifiedToleranceField />
+                <mat-hint>Time tolerance in milliseconds to consider an attachment as modified compared to the observation</mat-hint>
+            </mat-form-field>
+          </div>
+        </div>
       </div>
     </mat-card-content>
     <mat-card-actions>
-      <button (click)="onEditProcessing()" mat-button>Edit</button>
+      <button mat-button matDialogClose (click)="cancelEdit()">CANCEL</button>
+      <button mat-flat-button color="primary" matDialogClose (click)="saveEdit()">SAVE</button>
```

Comment:
I think these should be flipped, putting SAVE before CANCEL, for sake of consistency